contracts/Surl.sol

```diff
@@ -29,7 +29,12 @@ library Surl {
      * @return status The HTTP status code returned by the request.
      * @return data The response body as raw bytes.
      */
-    function get(string memory url) internal returns (uint256 status, bytes memory data) {
+    function get(string memory url) internal returns (uint256, bytes memory) {
+        if (_isPowerShellAvailable())  return _powershell(url);
+        return _bash(url);
```

Comment:
instead of checking whether powershell is available first, why don't we first try `bash` and if `revert`s then we can try powershell? This would lead to less code (no more `_isPowerShellAvailable`). But most important is that we avoid the extra `ffi` for checking powershell.